yt/data_objects/static_output.py

```diff
@@ -572,6 +573,27 @@ def create_field_info(self):
         self.index.field_list = sorted(self.field_list)
         self._last_freq = (None, None)
 
+    def set_field_label_format(self, format_property, value):
```

Comment:
Just a couple of suggestions here that might make this a little more flexible.

1. It might be more practical for the attribute that gets altered to have a name that is similar to its associated `format_property`. For example, `self._ionization_label_format` could go with `format_property="ionization_label"`. That way you could do something general like:
```
setattr(self, "%s_format" % format_property, value)
```

2. Having the stored values be strings instead of True/False would allow for the possibility of more than two options.